pkgs/by-name/co/copacetic/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+  installShellFiles,
+  copacetic,
+  testers,
+}:
+buildGoModule rec {
+  pname = "copacetic";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "project-copacetic";
+    repo = "copacetic";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-hvSbjkqrd//thUex2It31Z4Vrj1u07WEAQFAnWiPo6M=";
+  };
+
+  vendorHash = "sha256-eefYbB88wXQME8ehm/ieVBtOmmtxHkZSsjE05yeQ7Gw=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  CGO_ENABLED = "0";
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/project-copacetic/copacetic/pkg/version.GitVersion=${version}"
+    "-X main.version=${version}"
+  ];
+
+  checkFlags =
+    let
+      # Skip tests that require network access
+      skippedTests = [
+        "TestNewClient/custom_buildkit_addr"
+        "TestPatch"
+        "TestPlugins/docker.io"
+      ];
+    in
+    [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
+
+  postInstall = ''
+    mv $out/bin/copacetic $out/bin/copa
+    installShellCompletion --cmd copa \
+      --bash <($out/bin/copa completion bash) \
+      --fish <($out/bin/copa completion fish) \
+      --zsh <($out/bin/copa completion zsh)
+  '';
+
+  passthru.updateScript = nix-update-script { };
+  passthru.tests.version = testers.testVersion {
+    package = copacetic;
+    command = "copa --version";
```

Comment:
This is not needed, as it's the default behaviour for `testVersion`

Reply:
Updated. Thanks.